### ISSUE TITLE
Add a button to refresh each page

### ISSUE THREAD

**Issue by jjzhang10** [Mardaneus86/futwebapp-tampermonkey#174]
READ BEFORE YOU ADD A FEATURE REQUEST: Do not request autobuyer features, because they are considered as cheating. Issues regarding autobuyers will be closed immediately.

### Describe the solution you'd like
In order to avoid abusing futbin, could you add a button for manually refreshing?
It means that the user will determine whether to refresh one page of players.
### Suggestions for implementation
Add any other context, mockups or screenshots about the feature request here.

**Comment by johndundee21** [comment]
Yes I get block mostly because it refresh multiple time when it’s not necessary but if it was possible the price are updated via another server than futbin or giving lowest price since in the market + plus a button to get actual price
